README.md

```diff
@@ -42,6 +62,60 @@ $ pip3 install .
 Note: On some machines, you have to use `pip` instead of `pip3` but in most cases,
 you will want to use `pip3` if it's available for your platform.
 
+For regular use, we recommend taking the archived files from [our release page](https://github.com/cyberark/conjur-api-python3/releases).
+
+### Update from Ruby to Python CLI
+
+To ensure a smooth experience moving from the Ruby to Python CLI, we recommend that you either remove the Ruby CLI from your machine or rename it.
+
+To do so:
+
+1. Run `which conjur` to get the path of the Ruby CLI gem
+2. Navigate to the outputted path and either remove it or update it `conjur.v6`
+
+## Commands
+
+| Command  | Description                                              |
+| -------- | -------------------------------------------------------- |
+| authn    | - Log in and log out                                     |
+| host     | - Manage hosts                                           |
+| init     | - Initialize the Conjur configuration                    |
+| list     | - List all available resources belonging to this account |
+| policy   | - Manage policies                                        |
+| user     | - Manage users                                           |
+| variable | - Manage variables                                       |
+| whoami   | - Provide information about the current logged-in user   |
+
+##### `conjur authn`
+
+TBD
```

Comment:
is there a benefit to leaving all of these TBDs in the README now?

Reply:
Envisioned that this will look like what we have for our Ruby CLI docs and was waiting for final UX for this. Will remove in the meantime